### PR TITLE
Disconnect prometheus rules so they are picked up by mimir

### PR DESCRIPTION
**Pull request by QuentinBisson**
## Checklist

I have:

- [ ] Described why this change is being introduced
- [ ] Separated out refactoring/reformatting in a dedicated PR
- [ ] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

File: service/controller/resource/monitoring/prometheus/resource.go [hunk 1]
```diff
@@ -320,20 +296,6 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
...
 
-		prometheus.Spec.RuleSelector = &metav1.LabelSelector{
-			MatchExpressions: []metav1.LabelSelectorRequirement{
-				{
-					Key:      key.ClusterTypeKey,
-					Operator: metav1.LabelSelectorOpNotIn,
-					Values:   []string{"workload_cluster"},
-				},
-				{
-					Key:      key.TeamLabel,
```
Comment (marieroque) [inline service/controller/resource/monitoring/prometheus/resource.go:331]: Do we want to execute customer rules ?

File: service/controller/resource/monitoring/prometheus/resource.go [hunk 2]
```diff
@@ -320,20 +296,6 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 			key.EtcdSecret(config.Installation, cluster),
 		}
 
-		prometheus.Spec.RuleSelector = &metav1.LabelSelector{
```
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply service/controller/resource/monitoring/prometheus/resource.go:323]: Why would we want to do that with mimir in the future?

File: service/controller/resource/monitoring/prometheus/resource.go (hunk 1 above)
Comment (QuentinBisson) [reply service/controller/resource/monitoring/prometheus/resource.go:331]: Mimir will take care of rule evaluation

Comment (marieroque) [reply service/controller/resource/monitoring/prometheus/resource.go:331]: We have to wait for mimir before merging this PR, right ?

Comment (QuentinBisson) [reply service/controller/resource/monitoring/prometheus/resource.go:331]: Oh yes yes yes

Comment (marieroque) [reply service/controller/resource/monitoring/prometheus/resource.go:331]: So should you set this PR in draft for the moment ?

Comment (QuentinBisson) [reply service/controller/resource/monitoring/prometheus/resource.go:331]: I cannot

Comment (QuentinBisson) [reply service/controller/resource/monitoring/prometheus/resource.go:331]: Well I do not see the button

**Pull request closed without merging**